test/cosidela_test.go

```diff
@@ -83,31 +89,43 @@ func newDelaNode(t require.TestingT, path string, port int) dela {
 	require.NoError(t, err)
 
 	// mino
-	router := tree.NewRouter(minogrpc.NewAddressFactory())
-	addr := minogrpc.ParseAddress("127.0.0.1", uint16(port))
-
-	certs := certs.NewDiskStore(db, session.AddressFactory{})
-
-	fload := loader.NewFileLoader(filepath.Join(path, certKeyName))
-
-	keydata, err := fload.LoadOrCreate(newCertGenerator(rand.New(rand.NewSource(0)),
-		elliptic.P521()))
-	require.NoError(t, err)
-
-	key, err := x509.ParseECPrivateKey(keydata)
-	require.NoError(t, err)
-
-	opts := []minogrpc.Option{
-		minogrpc.WithStorage(certs),
-		minogrpc.WithCertificateKey(key, key.Public()),
+	var onet mino.Mino
+	switch kind {
+	case minoGRPC:
+		router := tree.NewRouter(minogrpc.NewAddressFactory())
+		addr := minogrpc.ParseAddress("127.0.0.1", uint16(port))
+
+		certs := certs.NewDiskStore(db, session.AddressFactory{})
+
+		fload := loader.NewFileLoader(filepath.Join(path, certKeyName))
+		keydata, err := fload.LoadOrCreate(newCertGenerator(rand.New(rand.NewSource(0)),
+			elliptic.P521()))
+		require.NoError(t, err)
+
+		key, err := x509.ParseECPrivateKey(keydata)
+		require.NoError(t, err)
+
+		opts := []minogrpc.Option{
+			minogrpc.WithStorage(certs),
+			minogrpc.WithCertificateKey(key, key.Public()),
+		}
+
+		onet, err = minogrpc.NewMinogrpc(addr, nil, router, opts...)
+		require.NoError(t, err)
+	case minoWS:
+		listen, err := ma.NewMultiaddr("/ip4/127.0.0.1/tcp/0/ws")
```

Comment:
why use `/0/` and not `port` here ?

Reply:
`/port/` should also work here. I'll change that.